Run-SystemTest.ps1

```diff
@@ -48,6 +48,7 @@ Process {
     $validateContractSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.ValidateContract"
     $codeFirstSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.CodeFirst"
     $generateContractSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.GenerateContract"
+    $generateContractWithFrontendSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.GenerateWithFrontend"
```

Comment:
We should really start duplicating projects and simply pass different parameters since it mostly those that changed ^^

Reply:
yup, changed to build test projects with extra args instead